vm/src/vm/runners/cairo_runner.rs

```diff
@@ -1102,6 +1107,32 @@ impl CairoRunner {
     pub fn get_program(&self) -> &Program {
         &self.program
     }
+
+    /// Return CairoRunner.layout
+    pub fn get_layout(&self) -> &CairoLayout {
+        &self.layout
+    }
```

Comment:
Does this need to be public? Also, does this need to return the whole (internal) `CairoLayout` structure or is the name enough?

Reply:
The whole structure, as it's used for writing the layout's parameters when it's a dynamic one.

Reply:
What about it being public? Could that be avoided?

Reply:
Nope it doesn't, thanks!